src/types/entities/QuoteHub.ts

```diff
@@ -0,0 +1,17 @@
+export type AccessToken = {
+    token: string
+}
+
+export type Categories = {
+    id: string
```

Comment:
type id correctly with `CategoryType`

Reply:
This id is for the DB info, for the relation between User and Category.
I just change the name, wich should be a CategoryType.